_studio/mfx_lib/shared/src/mfx_h264_encode_vaapi.cpp

```diff
@@ -1465,13 +1465,17 @@ mfxStatus VAAPIEncoder::CreateAuxilliaryDevice(
     };
 
     std::vector<VAConfigAttrib> attrs;
-    attrs.reserve(sizeof(attr_types) / sizeof(attr_types[0]));
+    auto AV = [&](VAConfigAttribType t) -> uint32_t& { return attrs[idx_map[t]].value; };
```

Comment:
too large closure as for me, I'd vote for
```
auto AV = [&attrs, &idx_map](VAConfigAttribType t)
```

Reply:
ok